src/http_rest_api_handler.cpp

```diff
@@ -524,9 +552,9 @@ Status HttpRestApiHandler::processInferKFSRequest(const HttpRequestComponents& r
     if (!gstatus.ok()) {
         return gstatus;
     }
+    std::set<std::string> binaryOutputs = getRequestedBinaryOutputs(grpc_request, grpc_response);
```

Comment:
The name states that binary outputs are inside this variable which is misleading. Name it according to what it holds, suggestion: `selectedOutputNamesForBinaryRepresentation`?